### ISSUE TITLE
about results

### ISSUE THREAD

**Issue by LARLSN** [aRI0U/RandLA-Net-pytorch#18]
hello,could you please provide your best result got by your code

**Comment by CharlesGaydon** [comment]
Gentle bump on this issue. It would be really helpful to know to which extent the performance are different than the ones in the original paper ! :)

**Comment by merazlab** [comment]
HI @LARLSN ,
on S3DIS dataset i will get this result and it is very differs from original one.

`=== EPOCH 99/100 ===
Training loss: 1.9831392	Validation loss: 680.1278906	
Accuracy      |     0 |     1 |     2 |     3 |     4 |     5 |     6 |     7 |     8 |     9 |    10 |    11 |    12 |    13 |    OA
Training:     | 0.927 | 0.091 | 0.316 | 0.962 | 0.671 | 0.000 | 0.000 | 0.041 | 0.781 | 0.000 | 0.000 | 0.000 | 0.000 | 0.000 | 0.607
Validation:   | 0.952 | 0.010 | 0.164 | 0.368 | 0.287 | 0.000 | 0.000 | 0.011 | 0.320 | 0.000 | 0.000 | 0.000 | 0.000 | 0.000 | 0.304
IoU           |     0 |     1 |     2 |     3 |     4 |     5 |     6 |     7 |     8 |     9 |    10 |    11 |    12 |    13 |  mIoU
Training:     | 0.764 | 0.055 | 0.081 | 0.906 | 0.466 | 0.000 | 0.000 | 0.009 | 0.218 | 0.000 | 0.000 | 0.000 | 0.000 | 0.000 | 0.322
Validation:   | 0.325 | 0.008 | 0.053 | 0.300 | 0.189 | 0.000 | 0.000 | 0.007 | 0.073 | 0.000 | 0.000 | 0.000 | 0.000 | 0.000 | 0.095
Time elapsed: 3 min 50 s

=== EPOCH 100/100 ===
Training loss: 1.8751107	Validation loss: 17.0391586	
Accuracy      |     0 |     1 |     2 |     3 |     4 |     5 |     6 |     7 |     8 |     9 |    10 |    11 |    12 |    13 |    OA
Training:     | 0.917 | 0.041 | 0.312 | 0.961 | 0.637 | 0.000 | 0.000 | 0.276 | 0.664 | 0.000 | 0.000 | 0.000 | 0.001 | 0.000 | 0.580
Validation:   | 0.487 | 0.017 | 0.128 | 0.745 | 0.562 | 0.000 | 0.000 | 0.231 | 0.108 | 0.000 | 0.000 | 0.000 | 0.000 | 0.000 | 0.317
IoU           |     0 |     1 |     2 |     3 |     4 |     5 |     6 |     7 |     8 |     9 |    10 |    11 |    12 |    13 |  mIoU
Training:     | 0.764 | 0.031 | 0.075 | 0.918 | 0.468 | 0.000 | 0.000 | 0.054 | 0.186 | 0.000 | 0.000 | 0.000 | 0.001 | 0.000 | 0.310
Validation:   | 0.290 | 0.013 | 0.055 | 0.346 | 0.350 | 0.000 | 0.000 | 0.045 | 0.023 | 0.000 | 0.000 | 0.000 | 0.000 | 0.000 | 0.119
Time elapsed: 3 min 51 s`

**Comment by xmilu0315** [comment]
> 
Hello, I and you encountered the same problem, excuse me, how do you finally give up?

**Comment by wangzihao77** [comment]
> 嗨@LARLSN，在S3DIS数据集上，我会得到这个结果，它与原始结果非常不同。>> `=== EPOCH 99/100 ===培训损失：1.9831392 验证损失：680.1278906准确性 | 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | OA培训： | 0.927 | 0.091 | 0.316 | 0.962 | 0.671 | 0.000 | 0.000 | 0.041 | 0.781 | 0.000 | 0.000 | 0.000 | 0.000 | 0.000 | 0.607验证： | 0.952 | 0.010 | 0.164 | 0.368 | 0.287 | 0.000 | 0.000 | 0.011 | 0.320 | 0.000 | 0.000 | 0.000 | 0.000 | 0.000 | 0.304IoU | 0 | 1| 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | mIoU培训： | 0.764 | 0.055 | 0.081 | 0.906 | 0.466 | 0.000 | 0.000 | 0.009 | 0.218 | 0.000 | 0.000 | 0.000 | 0.000 | 0.000 | 0.322验证： | 0.325 | 0.008 | 0.053 | 0.300 | 0.189 | 0.000 | 0.000 | 0.007 | 0.073 | 0.000 | 0.000 | 0.000 | 0.000 | 0.000 | 0.000 | 0.095时间：3分50秒>> === EPOCH 100/100 ===培训损失：1.8751107 验证损失：1.0391586准确性 | 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | OA培训： | 0.917 | 0.041 | 0.312 | 0.961 | 0.637 | 0.000 | 0.000 | 0.276 | 0.664 | 0.000 | 0.000 | 0.000 | 0.001 | 0.000 | 0.580验证： | 0.487 | 0.017 | 0.128 | 0.745 | 0.562 | 0.000 | 0.000 | 0.231 | 0.108 | 0.000 | 0.000 | 0.000 | 0.000 | 0.000 | 0.317IoU | 0 | 1| 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 | 13 | mIoU培训： | 0.764 | 0.031 | 0.075 | 0.918 | 0.468 | 0.000 | 0.000 | 0.054 | 0.186 | 0.000 | 0.000 | 0.000 | 0.001 | 0.000 | 0.310验证： | 0.290 | 0.013 | 0.055 | 0.346 | 0.350 | 0.000 | 0.000 | 0.045 | 0.023 | 0.000 | 0.000 | 0.000 | 0.000 | 0.000 | 0.000 | 0.119时间：351秒

哈哈，are you solved the problem? I meet the questions, too
